app/__tests__/home.test.ts

```diff
@@ -1,26 +1,7 @@
-import { UITestBuilder } from '@antiwork/shortest';
+import { test } from '@antiwork/shortest';
 
 interface loginButton {
   url: string;
 }
 
-define('Validate Dasboard is accessible by users', async () => {
-  afterAll(async () => {
-    console.log('Clearing DB after all tests');
-  });
-  beforeAll(async () => {
-    console.log('Clearing DB before all tests');
-  });
-  new UITestBuilder<loginButton>('/')
-    .test('Validate that users can access the dashboard')
-    
-    .given('baseUrl', { url: 'http://localhost:3000' })
```

Comment:
Should default to /, and otherwise be in the proper folder with a page.tsx or route.ts and then the route can be assumed due to filesystem-based routing of Nextjs. we could add a url option for tests that don't follow that format

(in the case of Shortest, we should move the tests to those places so it can serve as a really good demo.)